packages/slice-machine/server/src/resolveAliases.ts

```diff
@@ -0,0 +1,18 @@
+import path from "path";
```

Comment:
As this file is used by script and node server, we can put this file into the lib folder na ?

Reply:
good point, i didn't know where to put it.